File: PHP/Algorithms/Math/leastCommonMultiple.php
<?php

require_once 'greatestCommonDivisor.php';

/**
 * Calculates the least common multiple of two numbers
 *
 * @param int $firstNumber
 * @param int $secondNumber
 * @return int
 */
function leastCommonMultiple($firstNumber, $secondNumber)
{
    return abs($firstNumber * $secondNumber) / greatestCommonDivisor($firstNumber, $secondNumber);
}

File: C/Algorithms/Math/leastCommonMultiple.c
#include "greatestCommonMultiple.c"
#include <stdlib.h>

/**
 * Calculates the least common multiple of two numbers
 *
 * @param int firstNumber
 * @param int secondNumber
 * @return int
 */
int leastCommonMultiple(int firstNumber, int secondNumber)
{
    return abs(firstNumber * secondNumber) / greatestCommonDivisor(firstNumber, secondNumber);
}